OPM/opm-rule-onboarding.ps1

```diff
@@ -0,0 +1,540 @@
+###########################################################################
+#
+# NAME: OPM Rule Onboarding Utility
+#
+# AUTHOR:  Jeff Rechten
+# AUTHOR:  Assaf Miron
```

Comment:
No need for this :)
You can remove it

Reply:
Most of it is your code, I just changed a few things to make it work for OPM :)